libs/blocks/how-to/how-to.css

```diff
@@ -42,6 +42,11 @@
   line-height: 42px;
 }
 
+html[dir="rtl"] .how-to ol > li::before {
+  margin-left: 20px;
+  margin-right: inherit;
```

Comment:
I'd argue that this should be set to its implicit value, which in this case is `0`. Otherwise, adding margins to parents might lead to unexpected results here.

Reply:
Hi @overmyheadandbody. Using the initial keyword, rather than specifying 0, would work too, wouldn't it?

Reply:
Thanks @overmyheadandbody & @echampio-at-adobe. I will update the initial value with `0` and close this conversation.